.zenodo.json

```diff
@@ -2,23 +2,15 @@
   "title": "alpaka: Abstraction Library for Parallel Kernel Acceleration",
   "description": "The alpaka library is a header-only C++14 abstraction library for accelerator development. Its aim is to provide performance portability across accelerators through the abstraction (not hiding!) of the underlying levels of parallelism.",
   "creators": [
-    {
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden, LogMeIn Inc.",
-      "name": "Worpitz, Benjamin"
-    },
-    {
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
-      "name": "Widera, René",
-      "orcid": "0000-0003-1642-0459"
-    },
     {
       "name": "Bastrakov, Sergei",
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
       "orcid": "0000-0003-3396-6154"
     },
     {
-      "name": "Colgrove, Mat",
-      "affiliation": "NVIDIA"
+      "name": "Di Pilato, Antonio",
+      "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
```

Comment:
I would add CERN, or CMS. Best to ask @tonydp03.

Reply:
Well in principle I have a contract with CASUS/HZDR, so that's correct. I would also add CERN as additional affiliation - as an institute/organization, rather than the CMS experiment specifically.

Reply:
Thanks! Fixed.